client/src/components/App/Content/InvestigationForm/useInvestigationForm.ts

```diff
@@ -90,7 +91,20 @@ const useInvestigationForm = (): useInvestigationFormOutcome => {
             .catch(error => {
                 contactTypesLogger.error(`got errors in server result: ${error}`, Severity.HIGH);
             });
-    }
+    };
+
+    const fetchIdentificationTypes = () => {
+        const identificationTypesLogger = logger.setup('Fetching Identification Types');
+        identificationTypesLogger.info('launching request to get identification types', Severity.LOW);
+        axios.get('/investigationInfo/identificationTypes')
+            .then((result: any) => {
```

Comment:
why any?

Reply:
we always use any type for result because its the server respond and there are a lot of fields there so doing type to that it's hard

Reply:
yea but doesn't identificationTypes supposed to return the Store's type?